experiments/sidb_simulation/electrostatic_ground_state/runtime_analysis_bestagon_gates.cpp

```diff
@@ -100,14 +108,28 @@ int main()  // NOLINT
             tts_quicksim += stats.time_to_solution;
 
             instances += 1;
+            total_number_of_instances += 1;
             quicksim_accuracy_mean += stats.acc;
             quicksim_single_runtime += stats.mean_single_runtime;
         }
         quicksim_accuracy_mean = quicksim_accuracy_mean / static_cast<double>(instances);
+
+        total_runtime_exhaustive += runtime_exhaustive;
+        total_runtime_quickexact += runtime_quickexact;
+        average_accuracy_quicksim += quicksim_accuracy_mean;
+        total_single_rumtime_quicksim += quicksim_single_runtime;
+        total_tts_quicksim += tts_quicksim;
+
         simulation_exp(gate, instances, runtime_exhaustive, runtime_quickexact, quicksim_accuracy_mean,
                        quicksim_single_runtime, tts_quicksim);
         simulation_exp.save();
         simulation_exp.table();
     }
+
+    simulation_exp("Overall", total_number_of_instances, total_runtime_exhaustive, total_runtime_quickexact,
```

Comment:
```suggestion
    simulation_exp("Total", total_number_of_instances, total_runtime_exhaustive, total_runtime_quickexact,
```

Reply:
I thought the same, but is this appropriate for the average accuracy part?

Reply:
I think that is good! Thanks.

Reply:
You mean because it could invoke the notion of a sum?